doc/kubernetes/modules/ROOT/pages/running/metrics/keycloak_cluster.adoc

```diff
@@ -0,0 +1,27 @@
+= {project_name} cluster deployment
+:description: This document contains details of the metrics to monitor your {project_name} deployment's performance.
+:jgrp_application: {project_name}
+:clustered_caches: true
+
+{description}
+
+The deployment described in this document is for single-site deployments.
+In this architecture, {project_name} nodes creates cluster between themselves, used by the {ispn} clustered caches.
+
+If an external {jdgserver_name} is used, check xref:./keycloak_with_external_infinispan.adoc[] guide.
+
+include::partial$/running/metrics/kc_enable_metrics.adoc[leveloffset=+1]
+include::partial$/running/metrics/kc_http_metrics.adoc[leveloffset=+1]
+
+== Clustering and Network
+
+Multiple {project_name} nodes allows to distribute the load among them, but it requires communication between the nodes.
```

Comment:
"Deploying multiple {project_name} nodes allows load to be distributed amongst them, but this requires communication between the nodes."